src/AlgoliaSearch.js

```diff
@@ -546,6 +546,31 @@ AlgoliaSearch.prototype.assignUserID = function(data, callback) {
   });
 };
 
+/**
+ * Assign a array of userIDs to a cluster.
+ *
+ * @param {Array} data.userIDs The array of userIDs to assign to a new cluster
+ * @param {string} data.cluster The cluster to assign the user to
+ * @return {Promise|undefined} Returns a promise if no callback given
+ * @example
+ * client.assignUserIDs({ cluster: 'c1-test', userIDs: ['some-user-1', 'some-user-2'] });
+ */
+AlgoliaSearch.prototype.assignUserIDs = function(data, callback) {
```

Comment:
we should be using this batch as well for the assignUserId method right above this one, so it doesn't need that ugly "headers" argument we had to add specifically for it

Reply:
They're are different methods. The `client.assignUserIDs` doesn't allow to move userIds, and we are going to specify that on the docs.